README.md

```diff
@@ -121,6 +113,36 @@ Pylance provides users with the ability to customize their Python language suppo
         -   `true` (default)
         -   `false`
 
+-   `python.analysis.indexing`
+
+    - Used to specify whether Pylance should index installed third party libraries and user files to provide features such as auto-import, add import, workspace symbols, etc.
+    - Available values:
+        - `null` (default)
```

Comment:
I realize this isn't new (just moved), but what does null mean? What is the resulting behavior? Is it equivalent to true?